backend/config/__init__.py

```diff
@@ -95,5 +95,8 @@ def str_to_bool(value: str) -> bool:
 DISABLE_EMULATOR_JS = str_to_bool(os.environ.get("DISABLE_EMULATOR_JS", "false"))
 DISABLE_RUFFLE_RS = str_to_bool(os.environ.get("DISABLE_RUFFLE_RS", "false"))
 
+# FRONTEND
+UPLOAD_TIMEOUT = int(os.environ.get("UPLOAD_TIMEOUT", 600000))
```

Comment:
As discussed on Discord, this variable should be in seconds, to maintain consistency with `SCAN_TIMEOUT`. I think that's good enough for most users. If at some point we need millisecond precision, we can add `UPLOAD_TIMEOUT_MS`, which is evaluated only if `UPLOAD_TIMEOUT` is unset.

Reply:
Done :)